lib/goose/dbconf.go

```diff
@@ -121,9 +122,15 @@ func NewDBConf(dbDir, env string) (*DBConf, error) {
 		drv = imprt[i+1:]
 	}
 
-	open, _ := confGet(f, env, "open")
+	dsn, err := confGet(f, env, "dsn")
+	if _, ok := err.(*yaml.NodeNotFound); ok {
+		log.Println("WARNING: Database 'dsn' not specified. Please check ensure " +
+			"that you have a 'dsn' node set in your dbconf file. If you " +
+			"are not using a conf file please ensure that you have set " +
+			"the 'DB_DSN' environment variable.")
```

Comment:
Ah ok. I wasn't aware of that. Still trying to wrap my head fully around the project. What do you think about printing this for the drivers that require the dsn?